src/dir.targets

```diff
@@ -10,6 +10,15 @@
   <Target Name="GetAssemblyVersion"
           Returns="$(AssemblyVersion)"/>
 
+  <!-- These gets overwritten when we restore the build tools and import the below targets -->
+  <Target Name="Build" Condition="!Exists('$(ToolsDir)Build.Common.targets')" >
+    <CallTarget Targets="_RestoreBuildTools" />
+  </Target>
+  <!-- Set a default clean target -->
+  <Target Name="Clean">
```

Comment:
Good feedback.  I originally wanted to clean just specific libraries for the clean command, but found that you didn't always have sufficient behavior, so I resorted to the behavior which I previously noticed for the Clean command of just wiping the bin directory.  I think just having an empty Clean target is better, I may add an informative message in this scenario as well saying that nothing is actually happening.